packages/server/modules/automate/repositories/automations.ts

```diff
@@ -93,6 +93,28 @@ export async function getFullAutomationRevisionMetadata(
   }
 }
 
+export type InsertableAutomationFunctionRun = Omit<
+  AutomationFunctionRunRecord,
+  'id' | 'runId'
```

Comment:
If we're ignoring some params, date stuff, like created etc prob doesn't need to be part of the insert model

Reply:
done, simplified